scripts/sync.sh

```diff
@@ -41,6 +45,9 @@ candidates() {
 }
 
 pop() {
+    git config --global user.name "OpenShift Merge Robot"
+    git config --global user.email "openshift-merge-robot@users.noreply.github.com"
```

Comment:
I think this set up logic would live in the python script's logic?

Reply:
That makes total sense, I left a comment in the other PR that we should add an if statement or handle it in `sync.py`.